.changeset/ripe-lights-appear.md

```diff
@@ -0,0 +1,5 @@
+---
+"@credo-ts/openid4vc": minor
+---
+
+Add support for deferred credential issuance according to Draft 16 of the OpenID for Verifiable Credential Issuance specification.
```

Comment:
Do we also add support for Draft 16? And if so can we add a changeset for that

Reply:
I can quickly take a look and add support for that on `oid4vc`